src/components/popover/popover.jsx

```diff
@@ -0,0 +1,94 @@
+import PropTypes from 'prop-types';
+import React, {
+  forwardRef,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from 'react';
+import IconButton from '../icon-button/icon-button.jsx';
+import iconClear from '../../svgs/clear.svg';
+import styles from './popover.css';
+
+const Popover = forwardRef(function Popover(props, ref) {
+  const [open, setOpen] = useState(false);
+  const closeIconRef = useRef();
+  const backupRef = useRef();
+  const selfRef = ref || backupRef;
+
+  const onWindowClick = () => {
+    if (open) {
+      setOpen(false);
+    }
+  };
+
+  useEffect(() => {
+    window.addEventListener('click', onWindowClick);
+
+    if (closeIconRef.current && open) {
+      closeIconRef.current.focus();
+    }
+
+    if (!open) {
+      props.onClose();
+    }
+
+    return () => {
+      window.removeEventListener('click', onWindowClick);
+    };
+  }, [open]);
+
+  useImperativeHandle(selfRef, () => ({
+    get open() {
+      return open;
+    },
+    set open(value) {
+      setOpen(value);
+    },
+  }));
+
+  if (!open) {
+    return null;
+  }
+
+  return (
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+    <section
+      aria-labelledby="popover-heading"
```

Comment:
If a Popover can be left opened, this ID will be duplicated on the DOM. 

We will want to ensure a unique ID is generated on every instance of a Popover. 

Historically, we've been providing functionality by adding an `id` prop and letting the user provide us with a unique string -- and then we either use it or add to it.

e.g. 

```suggestion
      aria-labelledby={`${props.id}-popover-heading`}
```

Reply:
DRD is going to be amended to show that only one `Popover` can be open at a time.